charts/lakefs/templates/deployment.yaml

```
@@ -19,6 +19,16 @@ spec:
       labels:
         {{- include "lakefs.selectorLabels" . | nindent 8 }}
     spec:
+      {{- if eq ( include "lakefs.fullname" .) "treeverse/lakefs-enterprise" }}
```

Not sure this condition adds anything really but either way notice the fix. 
(please verify it works with `helm template`)
```suggestion
      {{- if eq ( include "lakefs.repository" .) "treeverse/lakefs-enterprise" }}
```